tests/cypress/support/e2e.js

```diff
@@ -15,24 +15,44 @@
 
 // Import commands.js using ES2015 syntax:
 
-require('@cypress/code-coverage/support');
+import './commands';
+import addContext from 'mochawesome/addContext';
+
+// // eslint-disable-next-line @typescript-eslint/no-var-requires
+// require('cypress-terminal-report/src/installLogsCollector')({
+//     xhr: {
+//         printHeaderData: true,
+//         printRequestData: true
+//     },
+//     enableExtendedCollector: true,
+//     collectTypes: ['cons:log', 'cons:info', 'cons:warn', 'cons:error', 'cy:log', 'cy:xhr', 'cy:request', 'cy:intercept', 'cy:command']
+// });
```

Comment:
Actually I don't know what it's used for, but it makes my whole tests crashes since I upgraded some frameworks. I had to remove but we may need to reactivate it - @Fgerthoffert  ?

Reply:
This is used to collect more logs while the tests are executed. It makes it easier to debug issues.
Do you know why it caused a crashes ?